.github/workflows/ci.yml

```diff
@@ -106,6 +106,13 @@ jobs:
           echo "::set-output name=version::$($xml.project.parent.version)"
           echo "::set-output name=class::$($xml.project.build.plugins.plugin.executions.execution.configuration.transformers.transformer.mainClass)"
         shell: pwsh
+      - name: Set tags
+        id: version
+        run: |
+          if [ ${{ steps.info.outputs.version }} != *"SNAPSHOT"* ]
+          then
+            echo "::set-output name=latestTag::'latest'"
+          fi
```

Comment:
question: why do we have this condition here, but then also below at line 128?

Reply:
i just saw a bug here, with calling `steps.info.version.latestTag` instead of steps.version.outputs.latestTag in line [128](https://github.com/ljubon/siembol/blob/93c14c9ec4af0d478cc30656248cfbc3f2ee3d90/.github/workflows/ci.yml#L128)

to answer your question, the idea behind this condition is to confirm should we have two tags for `stable` images (e.g `1.4.0` and `latest` tags)